bot/constants.py

```diff
@@ -24,6 +25,9 @@
 DISCORD_BOOST_LVL2_CAP = 15
 DISCORD_BOOST_LVL3_CAP = 30
 
+# Special Role IDs
+ROLE_ID_MODERATOR = 356080544670285825
```

Comment:
Is this for the PC master race server or for the university server? 
Otherwise I'd recommend to use these in env so we can have a debug env for the master race server and a prod env for the uni server (also for CHANNEL_ID_MODMAIL below).

Reply:
This are the IDs for our test server (PC Master Race).
Yeah we could do that.